src/pages/LoginAndRegister/RegisterSocial.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+	<www-page>
+		<div class="page-content">
+			<form
+				id="registerSocialTermsForm"
+				ref="registerSocialTermsForm"
+				class="promptForm"
+				@submit.prevent.stop="postRegisterSocialForm">
```

Comment:
You can also add `action="."` which makes it so that the submit event is fired when pressing 'enter' or submitting the form in some other way besides clicking the submit button

Reply:
Thanks, done with https://github.com/kiva/ui/commit/cf2b65dc1799d4278f3521a78b8d1c199c3b488e